lib/core/Canvas.js

```diff
@@ -221,18 +224,34 @@ Canvas.prototype.getDefaultLayer = function() {
  *
  * @returns {Snap<SVGGroup>}
  */
-Canvas.prototype.getLayer = function(name) {
+Canvas.prototype.getLayer = function(name, index) {
 
   if (!name) {
     throw new Error('must specify a name');
   }
 
   var layer = this._layers[name];
+
   if (!layer) {
```

Comment:
Also clarify how the interaction is if two layers are created without index _or_ with the same index.